cel/options.go

```diff
@@ -374,6 +378,14 @@ func EvalOptions(opts ...EvalOption) ProgramOption {
 	}
 }
 
+// CallCostEstimator enables actual runtime cost estimator and overwrites callCostEstimator
```

Comment:
Instead of "overwrites callCostEstimator" (which is an internal detail users are not expected to understand), how about "CallCostEstimator registers a ActualCostEstimator that can optionally provide a runtime cost estimate for any function calls. This enables runtime costs to be assigned to calls function library extensions." ?